src/others/components/Map.tsx

```diff
@@ -15,6 +54,12 @@ export function Map() {
   const [lat] = useState(ukraineCenterCoords.lat);
   const [zoom] = useState(4);
```

Comment:
*non blocking*
I don't believe `lng`, `lat` nor `zoom` need to be stateful.
Defining
```typescript
const initialView= {
    lat: 48.4501071,
    lng: 30.5240501,
    zoom: 4,
  };
```
and passing it directly to `initialViewState` should do the job.

Reply:
Done